src/range-slider/range-slider.ts

```diff
@@ -59,7 +59,7 @@ export class NgAisRangeSlider extends BaseWidget {
 
   public ngOnInit() {
     this.createWidget(connectRange, {
-      attributeName: this.attribute,
+      attribute: this.attribute,
       max: parseNumberInput(this.max),
       min: parseNumberInput(this.min),
       precision: parseNumberInput(this.precision),
```

Comment:
Inside the `rangeInput` widget we dropped the call to `parseNumberInput`. Careful because it might be possible that Angular provides the value as string rather than number. Worth checking it with Storybook.

Reply:
I put it back. It's safer to have it.